crates/ruff_python_formatter/src/other/except_handler_except_handler.rs

```diff
@@ -36,56 +37,65 @@ impl FormatNodeRule<ExceptHandlerExceptHandler> for FormatExceptHandlerExceptHan
         item: &ExceptHandlerExceptHandler,
         f: &mut PyFormatter,
     ) -> FormatResult<()> {
-        let ExceptHandlerExceptHandler {
-            range: _,
-            type_,
-            name,
-            body,
-        } = item;
+        format_except_handler(item, self.except_handler_kind, true, f)
+    }
+}
 
-        let comments_info = f.context().comments().clone();
-        let dangling_comments = comments_info.dangling(item);
+pub(crate) fn format_except_handler(
+    item: &ExceptHandlerExceptHandler,
+    except_handler_kind: ExceptHandlerKind,
+    last_suite_in_statement: bool,
+    f: &mut PyFormatter,
+) -> FormatResult<()> {
```

Comment:
Nit: Any specific reason why you decide to introduce a `format` function for except handlers rather than adding a field to the options? I'm fine with either, I'm mainly surprised that you went with a format function here, but decided to introduce an `Options` struct for `match_case`

Reply:
It's now `Options` everywhere.